spec/requests/stories/feeds_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe "Stories::FeedsIndex", type: :request do
+  let(:title) { "My post" }
+  let!(:article) { create(:article, title: title, featured: true) }
+
+  describe "GET feeds index" do
+    let(:response_json) { JSON.parse(response.body) }
+    let(:response_article) { response_json.first }
+
+    it "renders article list as json" do
+      get "/stories/feed", headers: headers
+      expect(response.content_type).to eq("application/json")
+      expect(response.body).to include(article.title)
+      expect(response_article["title"]).to eq title
+      expect(response_article["class_name"]).to eq "Article"
```

Comment:
I'm testing a method and an attribute in the output, but of course that's far from complete. What's everyone's opinion? Dynamic attributes on `article` make testing the output pretty tedious, but an article fixture seems too brittle.

Reply:
This is really just doing a sanity check to see we're getting an array of articles back. Checking for a few properties which seems OK. Other things to consider may be the order of the articles returned etc. but those are tests for the feed service which appears to already have some tests related to that, e.g. not including a low score, articles from last 6 hours etc.

Thoughts?

Reply:
Is there a reason for the class name? Is it because the feed might contain different type of models in the future?

Reply:
Exactly @nickytonline: I just wanted to do a sanity test that JSON is being properly rendered here, and test a few attributes. Testing _which_ articles are included is done in the test for `Articles::Feed`

Reply:
@rhymes: I wanted to test the output of an `Article` attribute, and a method. Class name was the most stable one to test.